src/rpc/server.cpp

```diff
@@ -468,6 +468,11 @@ std::string JSONRPCExecBatch(const UniValue& vReq)
 
 UniValue CRPCTable::execute(const std::string &strMethod, const UniValue &params) const
 {
+    auto span = TracingSpan("info", "rpc", "Execute", "method", strMethod.c_str());
+    auto spanGuard = span.Enter();
```

Comment:
These two lines don't seem to be necessary. I think they're for when one thread can start a timer and possibly different thread ends the timing interval. But @strd will know for sure.